client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.ts

```diff
@@ -588,6 +590,64 @@ export class MotionDetailComponent extends BaseModelContextComponent implements
         );
     }
 
+    private requestUpdates(): void {
+        this.requestUserUpdates();
```

Comment:
Please merge those requests together: Each update request will issue a real request to the server. Maybe you can collect follows together and make a single request then.